packages/components/src/components/tick-progress/tick-progress.jsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Tick = ({ is_on }) => {
+    return (
+        <div
+            className={classNames('dc-tick-progress__tick', {
+                'dc-tick-progress__tick--on': is_on,
+            })}
+        />
+    );
+};
+
+const TickProgress = ({ className, rows, columns, size, value }) => {
+    return (
+        <div
+            className={classNames('dc-tick-progress', className)}
+            style={{
+                gridTemplateRows: `repeat(${rows}, 1fr)`,
+                gridTemplateColumns: `repeat(${columns}, 1fr)`,
+            }}
+        >
+            {new Array(size).fill(null).map((item, index) => {
+                return <Tick is_on={index < value} key={index} />;
```

Comment:
should this be `index + 1 < value` ?

Reply:
@kelly-binary No. because `index` starts from 0 and `value` starts from 1